Component/Component/View/EmptyView/FamiliEmptyView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  FamiliEmptyView.swift
+//  Component
+//
+//  Created by Owen Prasetya on 17/11/20.
+//
+
+import UIKit
+
+/** Empty View
+ 
+    Create an empty view with predefined positioning.
+    How to use:
+    * Using xib
+        * * Drag and drop a UIView and change the class to FamiliEmptyView, put it inside a container view.
+    * Programatically
+        * Init a new empty view, with functions to set the image name and label text of the view.
+        * Use setLabel function to change the text into the desired text.
+        * Use setImage function to change the image into the desired image..
+*/
+
+@IBDesignable public class FamiliEmptyView: UIView {
+    // MARK: - Alias
+    typealias CommonProperty = FamiliEmptyViewConstant.CommonProperties
+    
+    // MARK: - Property
+    @IBInspectable var labelText: String = "" {
+        didSet {
+            emptyViewLabel.text = labelText
+            setupLabel()
+        }
+    }
+    
+    @IBInspectable var imageName: String = "" {
+        didSet {
+            guard let image = UIImage(named: imageName) else { return }
```

Comment:
use UIImage(named: , in: ,compatibleWith:) initialization to make image visible

Reply:
updated